tests/ceph_installer/test_cephadm.py

```diff
@@ -49,6 +57,75 @@
 )
 
 
+def get_host_osd_map(cls):
+    out, err = cls.shell(args=["ceph", "osd", "tree"])
+    osd_string = out
+    osd_dict = {}
+    for osd_str in osd_string.split("host")[1:]:
+        osd = [i.strip() for i in osd_str.splitlines()]
+        osd_dict[osd[0]] = list()
+        for i in osd[1:]:
+            temp = i.split()
+            for x in temp:
+                if "osd" in x:
+                    osd_dict[osd[0]].append(x)
+    return osd_dict
+
+
+def validate_log_file_after_enable(cls):
+    """
+    Method to verify generation of log files in default log directory when logging not enabled
+    Args:
+        cls: cephadm instance object
+
+    Returns:
+        string
+    """
+    out, err = cls.shell(
+        args=["ceph", "config", "set", "global", "log_to_file", "true"]
+    )
+    out, err = cls.shell(args=["ceph", "fsid"])
+    fsid = out.strip()
+    log_file_path = os.path.join("/var/log/ceph", fsid)
+    roles_to_validate = ["mon", "mgr", "osd"]
```

Comment:
IMO, Most daemons produces log files, So its good to go with current daemons and its respective log file.
Hope daemon name and log file name is almost same.

Reply:
when we enable logging using the command 
ceph config set global log_to_file true
as we discussed for this task, only MON, MGR, OSD, RGW and MDS logs are getting enabled. I will add validation for these roles.